blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1076,7 +1076,7 @@ async fn sync_messages_check_state<
     invalid_block_strategy: InvalidBlockStrategy,
 ) -> Result<(), TipsetRangeSyncerError<C>> {
     // Sync the messages for one or many tipsets @ a time
-    const REQUEST_WINDOW: usize = 8;
+    const REQUEST_WINDOW: usize = 4;
```

Comment:
Any reason to restrict the window to half size here?

Reply:
This is the related optimization: https://github.com/ChainSafe/forest/pull/1943

Reply:
>- Reduce chain exchange batch size for message lookup to reduce bundled message size.

The larger chain exchange response is, the more we try cbor-decoding it partially (with failure)

Reply:
Ah ok, get it now 👍